api/proto/teleport/okta/v1/okta_service.proto

```diff
@@ -44,6 +44,10 @@ service OktaService {
   rpc CreateOktaAssignment(CreateOktaAssignmentRequest) returns (types.OktaAssignmentV1);
   // UpdateOktaAssignment updates an existing Okta assignment resource.
   rpc UpdateOktaAssignment(UpdateOktaAssignmentRequest) returns (types.OktaAssignmentV1);
+  // UpdateOktaAssignmentActionStatuses will update the statuses for all actions in an Okta assignment if the
+  // status is a valid transition. If a transition is invalid, it will be logged and the rest of the action statuses
+  // will be updated if possible.
```

Comment:
nit: Since we aren't logging this to the audit log, I would omit this part. logging to stdout seems like more of an implementation detail that isn't relevant here and can be confusing.
```suggestion
  // status is a valid transition. Invalid transitions will be skipped.
```